src/alpaka/AlpakaCore/alpakaConfigAcc.h

```diff
@@ -116,4 +116,11 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
   using AlpakaDeviceSubView = alpaka::ViewSubView<Device, TData, Dim1D, Idx>;
 }  // namespace ALPAKA_ACCELERATOR_NAMESPACE
 
+namespace cms::alpakatools {
+  template <>
+  struct DevType<ALPAKA_ACCELERATOR_NAMESPACE::Event> {
+    using type = ALPAKA_ACCELERATOR_NAMESPACE::Device;
+  };
```

Comment:
I guess if we end up using this, we should ask Alpaka developers to add it centrally ?

Reply:
Done in https://github.com/alpaka-group/alpaka/issues/1428